assignment3/assignment_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:test/test.dart';
+import 'functions.dart';
+import 'graph.dart';
+
+void main() {
+  test('i want to test graph Class', () {
```

Comment:
Follow better naming convention for your test cases.

Reply:
i have changed the naming convention